tutorials/animation/2d_skeletal_deform.rst

```diff
@@ -0,0 +1,240 @@
+.. _doc_2d_skeletons:
+
+2D skeletons
+============
+
+.. warning::
+
+    There are known issues with 2D skeletons on mobile and web platforms with the GLES2 renderer. We
+    recommend using the GLES3 renderer if your project relies on Skeleton2D for now.
+
+Introduction
+------------
+
+When working with 3D, skeletal deforms are common for characters and creatures
```

Comment:
Sometimes you use the term "skeletal deforms" and sometimes "skeletal deformations". I would prefer "skeletal deformations"